modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/display/context/CategoryCPDisplayLayoutDisplayContext.java

```diff
@@ -87,6 +96,46 @@ public String getAddCategoryDisplayPageURL() throws Exception {
 		return portletURL.toString();
 	}
 
+	public AssetCategory getAssetCategory(long assetCategoryId)
+		throws PortalException {
+
+		return _assetCategoryLocalService.getAssetCategory(assetCategoryId);
+	}
+
+	public String getCategorySelectorURL(RenderResponse renderResponse) {
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		try {
+			PortletURL portletURL = PortletProviderUtil.getPortletURL(
+				httpServletRequest, AssetCategory.class.getName(),
+				PortletProvider.Action.BROWSE);
+
+			if (portletURL == null) {
+				return null;
+			}
+
+			List<AssetVocabulary> vocabularies =
+				AssetVocabularyServiceUtil.getGroupVocabularies(
+					themeDisplay.getCompanyGroupId());
+			portletURL.setParameter(
+				"eventName", renderResponse.getNamespace() + "selectCategory");
+			portletURL.setParameter("singleSelect", "true");
+			portletURL.setParameter(
+				"vocabularyIds",
+				ListUtil.toString(
+					vocabularies, AssetVocabulary.VOCABULARY_ID_ACCESSOR));
+			portletURL.setWindowState(LiferayWindowState.POP_UP);
```

Comment:
@ivicac new line here.
```suggestion
			List<AssetVocabulary> vocabularies =
				AssetVocabularyServiceUtil.getGroupVocabularies(
					themeDisplay.getCompanyGroupId());

			portletURL.setParameter(
				"eventName", renderResponse.getNamespace() + "selectCategory");
			portletURL.setParameter("singleSelect", "true");
			portletURL.setParameter(
				"vocabularyIds",
				ListUtil.toString(
					vocabularies, AssetVocabulary.VOCABULARY_ID_ACCESSOR));
			portletURL.setWindowState(LiferayWindowState.POP_UP);
```